### ISSUE TITLE
Class "ResponseSourcesObject" does not map JSON properties correctly

### ISSUE THREAD

**Issue by code-bringer** [factset/enterprise-sdk#17]
- **Package Name**: FactSet.SDK.GlobalFilings
- **Package Version**: 1.0.1
- **Package Language**: C#
- **Operating System**:  Windows

**Describe the bug**
.NET class mapping to JSON object returned by API endpoint is not correct.

**To Reproduce**

```csharp
var apiInstance = new FilingsAPIApi(config);
ResponseSources sources = apiInstance.GetSources();
ResponseSourcesObject source = sources.Data[0];
Assert.NotNull(source.SourcesObject);
```

**Expected behavior**
The value of property "SourcesObject" of type "ResponseSourcesObject" should not be equal to null.

**Additional context**
I think there is an issue in .NET mapping to JSON object.
According to this [documentation ](https://developer.factset.com/api-catalog/global-filings-api#sdkLibrary)the response object should have the following structure:
```json
{
    "data": [{
            "source": "EDG",
            "description": "EDGAR"
        }, {
            "source": "EDJ",
            "description": "EDINET"
        }
    ]
}
```

But current expected of .NET code of JSON structure is following:
```json
{
    "data": [{
            "sourcesObject": {
                "source": "EDG",
                "description": "EDGAR"
            }
        }, {
            "sourcesObject": {
                "source": "EDJ",
                "description": "EDINET"
            }
        }
    ]
}
```

**Comment by eschmidtfds** [comment]
Hi @code-bringer thanks for filing this issue!
We've opened an internal support issue and will report back when a fix is available!

**Comment by eschmidtfds** [comment]
Hi @code-bringer , an SDK fix just went out to nuget: v1.0.2 of the Global Filings SDK.
Please reopen this issue, if that did not fix your bug!
